supporter-product-data/src/main/scala/com/gu/model/zuora/response/MinimalZuoraSubscription.scala

```diff
@@ -0,0 +1,28 @@
+package com.gu.model.zuora.response
+
+import com.gu.supporterdata.model.ContributionAmount
+import io.circe.Decoder
+import io.circe.generic.semiauto.deriveDecoder
+
+case class MinimalZuoraSubscription(ratePlans: List[RatePlan]) {
+  def contributionAmount = for {
+    ratePlans <- ratePlans.headOption
+    charges <- ratePlans.ratePlanCharges.headOption
+  } yield ContributionAmount(charges.price, charges.currency)
+}
+
+case class RatePlan(ratePlanCharges: List[RatePlanCharge])
+
+case class RatePlanCharge(price: BigDecimal, currency: String)
+
+object MinimalZuoraSubscription {
+  implicit val decoder: Decoder[MinimalZuoraSubscription] = deriveDecoder
+}
+
+object RatePlan {
+  implicit val decoder: Decoder[RatePlan] = deriveDecoder
+}
+
+object RatePlanCharge {
+  implicit val decoder: Decoder[RatePlanCharge] = deriveDecoder
+}
```

Comment:
personally I'd put all these in the companion object of ZuoraSubscriptionService to keep them together, as this is pretty specific to the situation